pkg/lang/ir/supervisor.go

```diff
@@ -28,11 +28,14 @@ import (
 const (
 	horustTemplate = `
 name = "%[1]s"
-command = "%[2]s"
-stdout = "/var/logs/%[1]s_stdout.log"
-stderr = "/var/logs/%[1]s_stderr.log"
+command = """
+%[2]s
+"""
+stdout = "/var/log/horust/%[1]s_stdout.log"
+stderr = "/var/log/horust/%[1]s_stderr.log"
 user = "${USER}"
 working-directory = "${%[3]s}"
+%[4]s
```

Comment:
I wonder if this is too flexible. Currently, we only need the `start-after`.

Maybe we can switch to a TOML tool to generate it if it becomes more complex in the future.

Reply:
We can refactor this once we need more controls here.